src/IIIFPresentation/API/Helpers/IPathGenerator.cs

```diff
@@ -0,0 +1,58 @@
+﻿using API.Converters;
+using Models.Database.Collections;
+using Models.Database.General;
+
+namespace API.Helpers;
+
+public interface IPathGenerator
```

Comment:
Worth adding xml comments to all for consistency (or removing if obvious what's happening)?